Javascript/searchUppercase.js

```diff
@@ -0,0 +1,29 @@
+/*
+대문자 찾기
+한 개의 문자열을 입력받아 해당 문자열에 알파벳 대문자가 몇 개 있는지 알아내는 프로그램 
+을 작성하세요.
+
+- 입력설명
+첫 줄에 문자열이 입력된다. 문자열의 길이는 100을 넘지 않습니다. 
+- 출력설명
+첫 줄에 대문자의 개수를 출력한다. 
+- 입력예제 1
+KoreaTimeGood 
+- 출력예제 1
+3
+*/
+
+const solution = (str) => {
+  let count = 0;
+  if (str.length > 100) throw new Error("문자열의 길이는 100을 넘지 않습니다.");
+  for (s of str) {
+    const asciiCode = s.charCodeAt();
+    if (asciiCode >= 65 && asciiCode <= 90) {
```

Comment:
```suggestion
    if (s >= "A" && s <= "Z") {
```
로 수정하는 것이 A~Z라는 의미를 한 눈에 파악할 수 있어서 더 좋을 것 같네요.

Reply:
😮 헉 저렇게 쉬운 방법이...